main.js

```diff
@@ -160,4 +160,142 @@ const inventory = [
+
+/* Opdracht 2 */
+
+/* 2a */
+const tvTypes = inventory.map((television) => {
+    return television.type;
+});
+
+/* 2b */
+const televisionsSoldOut = inventory.filter((soldTelevision) =>{
+  return soldTelevision.originalStock === soldTelevision.sold;
+});
+
+/* 2c */
+const televisionsHasAmbilight = inventory.filter((ambilightTelevision) =>{
+  return ambilightTelevision.options.ambiLight;
+});
+
+/* 2d */
+//const sortedTelevisions = inventory.sort((televisieA, televisieB) => {
+  //return televisieA.price - televisieB.price;
+//});
+
+/* Opdracht 3 */
+
+/* 3a */
+function revenueTarget(televisions){
+  let amount = 0;
+  for(const television of televisions) {
+      amount = amount + television.originalStock;
+  }
+
+  let targetRevenueWrapper = document.createElement("span");
+  let targetRevenue = document.createTextNode(amount.toString());
+  targetRevenueWrapper.appendChild(targetRevenue);
+  document.getElementById("revenue_t").appendChild(targetRevenueWrapper);
+  document.getElementById("revenue_t").style.backgroundColor="blue";
+}
+
+revenueTarget(inventory);
+
+/* 3b */
+function salesSoFar(televisions) {
+  let sales = 0;
+
+  for (const television of televisions) {
+    sales = sales + (television.price*television.sold);
+  }
+
+  let salesWrapper = document.createElement("span");
+  let salesAmount = document.createTextNode(sales.toString());
+  salesWrapper.appendChild(salesAmount);
+  document.getElementById("to-sell").appendChild(salesWrapper);
+  document.getElementById("to-sell").style.backgroundColor="green";
+}
+
+salesSoFar(inventory);
+
+/* Opdracht 4 */
+function showTvTypes(tvTypes, limit){
+
+    for(let i=0; i<limit; i++){
+        let linebreak = document.createElement("br");
+        let output = document.createTextNode(tvTypes[i]);
+        document.getElementById("output-tv-types").appendChild(output);
+        document.getElementById("output-tv-types").appendChild(linebreak);
+    }
+}
+
+//const twoTelevisionTypes = showTvTypes(tvTypes, 2);
+
+/* Opdracht 5 */
+
+/* 5a */
+function showTelevisionDetails(televisionDetails){
+    return televisionDetails.brand + televisionDetails.type + " - " + televisionDetails.name + " " + formatScreenSize(televisionDetails.availableSizes) + " \r\n " ;
+}
+
+//showTelevisionDetails();
+
+/* 5b */
+function formatPrice(value){
+    const output = "€" + value + ",-";
+    return output;
+}
+
+/* 5c */
+function formatScreenSize(television){
+    let output="";
+
+      for(const screenSize of television){
+          output += screenSize + " inch (" + Math.ceil(screenSize*2.54) + " cm)";
+          if(television.length>1 && (television.length -1 !== television.indexOf(screenSize))) output += " | ";
+      }
+    return output;
+}
+
+/* 5d */
+function wrapLayout(television){
+    const wrapTelevisionLayout = document.createElement("div");
+    wrapTelevisionLayout.setAttribute("Id", "wrapTvLayout");
+    wrapTelevisionLayout.setAttribute('style', 'white-space: pre;');
```

Comment:
Dit had je in principe in een css class kunnen zetten, zodat de scheiding tussen javascript en css duidelijker en makkelijker te onderhouden blijft